recipes/fltk/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "fltk" %}
+{% set version = "1.3.4" %}
+{% set longversion = "1.3.4-2" %}
+{% set md5 = "69128be72365cbedd1d380739f0164e3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ longversion }}-source.tar.gz
+  url: http://fltk.org/pub/{{ name }}/{{ version }}/{{ name }}-{{ longversion }}-source.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - zlib
+    - jpeg
+    - libpng
+    - python  # [win] to try to force vc >= 14
```

Comment:
instead of doing this to force ``vc>=14``, please do something like that in your build section:

```
build:
  number: 0
  skip: True [win and vc<14]
```

this will skip the build for windows and VC less than 14